package.json

```diff
@@ -47,6 +47,5 @@
   "devDependencies": {
     "bestikk-log": "0.1.0",
     "expect.js": "0.3.1"
-  },
-  "readme": "# Reveal.js converter for Asciidoctor\n\n[Asciidoctor reveal.js](https://github.com/asciidoctor/asciidoctor-reveal.js) is a converter for [Asciidoctor](https://github.com/asciidoctor/asciidoctor) and [Asciidoctor.js](https://github.com/asciidoctor/asciidoctor.js) that transforms an AsciiDoc document into an HTML5 presentation designed to be executed by the [reveal.js](http://lab.hakim.se/reveal-js/) presentation framework.\n\nThis is the npm module. For setup instructions and the AsciiDoc syntax to use to write a presentation see the module's documentation at [https://github.com/asciidoctor/asciidoctor-reveal.js](https://github.com/asciidoctor/asciidoctor-reveal.js).\n"
```

Comment:
I believe this field is not used anymore since we now have a `README.md` file.
It's also easier to edit/maintain a Markdown file than an inlined text content inside a json file.